### DISCUSSION TITLE
When using the mitmproxy proxy to intercept streaming traffic, the live video is not being displayed. For example, when capturing live broadcasts from Douyin (TikTok).

### DISCUSSION THREAD

**Discussion by Boris-code** [mitmproxy/mitmproxy#6593] [Intercept Everything!]
#### Problem Description
When using the mitmproxy proxy to intercept streaming traffic, the live video is not being displayed. For example, when capturing live broadcasts from Douyin (TikTok).

#### System Information
Paste the output of "mitmproxy --version" here.
version:10.2.0

**Comment by Prinzhorn** [comment]
I've moved this to discussions. If you think this is a bug in mitmproxy, then file a bug report and fill out the full template.

**Reply by Boris-code** [reply]
![20240109151957](https://github.com/mitmproxy/mitmproxy/assets/14087023/52304d6d-ac9d-418f-893c-1c5f2f942a3c)

Open the regular proxy on my computer
```
mitmweb --mode=regular@18089 --ssl-insecure --set block_global=false --showhost --set confdir=./conf
```


set iptables rules in phone
```
sysctl -w net.ipv4.ip_forward=1
ip6tables -t mangle -A OUTPUT -m owner --uid-owner u0_a269 -j DROP
iptables -t nat -A OUTPUT -p tcp -m owner --uid-owner u0_a269 -j DNAT --to 192.168.8.139:18089
iptables -t nat -A OUTPUT -p udp -m owner --uid-owner u0_a269 -j DNAT --to 192.168.8.139:18089
```

FLV is a live video stream format. It can be well forwarded on Charles, maintaining a long connection, but it disconnects on mitmproxy.

**Reply by Prinzhorn** [reply]
Can this be reproduced on a desktop as well (would be much easier)? What website do I need to open and what needs to be done to reproduce the error?

**Comment by sdualasijia** [comment]
you can pass --set stream_large_bodies=1 when using the mitmproxy

**Reply by mhpsy** [reply]
thank you very much, bro
